src/cmd/api/controllers/video_info_handler.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Sogilis/Voogle/src/pkg/uuidgenerator"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/db/dao"
+	jsonDTO "github.com/Sogilis/Voogle/src/cmd/api/dto/json"
+)
+
+type VideoGetInfoHandler struct {
+	MariadbClient *sql.DB
+	UUIDGen       uuidgenerator.IUUIDGenerator
+}
+
+// VideoGetInfoHandler godoc
+// @Summary Get video informations
+// @Description Get video informations
+// @Tags informations
+// @Accept plain
+// @Produce plain
+// @Param id path string true "Video ID"
+// @Success 200 {Json} Video Infos
+// @Failure 400 {object} object
+// @Failure 404 {object} object
+// @Failure 500 {object} object
+// @Router /api/v1/videos/{id}/info [get]
+func (v VideoGetInfoHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	log.Debug("GET VideoGetInfoHandler - parameters ", vars)
+
+	id, exist := vars["id"]
+	if !exist {
+		log.Error("Missing video id")
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	if !v.UUIDGen.IsValidUUID(id) {
+		log.Error("Invalid id")
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	video, err := dao.GetVideo(r.Context(), v.MariadbClient, id)
+	if err != nil {
+		log.Error("Cannot found video : ", err)
+		if errors.Is(err, sql.ErrNoRows) {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		return
+	}
+
+	videoInfo := jsonDTO.VideoToInfoJson(video)
+	payload, err := json.Marshal(videoInfo)
```

Comment:
Maybe we can do : `json.NewEncoder(w).Ecnore(videoInfo)` and with this we remove a feat lines